tests/microgrid/test_microgrid_api.py

```diff
@@ -82,23 +83,39 @@ def connections(self) -> list[list[Connection]]:
         ]
         return connections
 
+    # ignore mypy: Untyped decorator makes function "components" untyped
+    @pytest.fixture
+    def metadata(self) -> Metadata:
+        """Fetch the microgrid metadata.
+
+        Returns:
+            the microgrid metadata.
+        """
+        return Metadata(
+            microgrid_id=8,
+            location=Location(latitude=52.520008, longitude=13.404954),
+        )
```

Comment:
I would probably also mock the `frequenz.sdk.microgrid.metadata._timezone_finder` object here, so you don't depend on really using the external library, but I guess if it keeps a local DB it should be fine too as it is. You just have to have in mind that you are indirectly testing the `timezonefinder` package if you use it in the tests, a failure in the resolution might not indicate a problem with our code necessarily.

Reply:
I agree it makes sense to mock it because otherwise there will be some I/O going on (opening files) if it uses the `timezonefinder` package and these are unit tests

Reply:
done